### PR TITLE
Update alerting_notification_channels.md

### LINKED ISSUE #19296: Fix Alerting Notification Channels API stacked headings
[Alerting Notification Channels API](https://grafana.com/docs/http_api/alerting_notification_channels/) has an h1 followed by an h2.

Write an introduction to this topic to go between the headings.

### PR DESCRIPTION
**Pull request by oddlittlebird**
Minor edits and fixed stacked heading issue.

**Which issue(s) this PR fixes**:

Fixes #19296

**Special notes for your reviewer**: The intro would be better if we could explain more fully to the user what this API is for and why they might want to use it. If you provide that information, then I will wordsmith it.

### CODE REVIEW

File: docs/sources/http_api/alerting_notification_channels.md [hunk 1]
```diff
@@ -11,15 +11,16 @@ parent = "http_api"
...
 
 The identifier (id) of a notification channel is an auto-incrementing numeric value and is only unique per Grafana install.
 
 The unique identifier (uid) of a notification channel can be used for uniquely identify a notification channel between
 multiple Grafana installs. It's automatically generated if not provided when creating a notification channel. The uid
-allows having consistent URL's for accessing notification channels and when syncing notification channels between multiple
-Grafana installs, see [alert notification channel provisioning](/administration/provisioning/#alert-notification-channels)
-for more information.
+allows having consistent URLs for accessing notification channels and when syncing notification channels between multiple
+Grafana installations, refer to [alert notification channel provisioning](/administration/provisioning/#alert-notification-channels).
```
Comment (marefr) [inline docs/sources/http_api/alerting_notification_channels.md]:
```suggestion
Grafana installations, refer to [alert notification channel provisioning]({{< relref "../administration/provisioning.md#alert-notification-channels" >}}).
```

**Review by marefr — DISMISSED** [review]
Added suggestion. Otherwise looks good.

**Comment by oddlittlebird** [comment]
@marefr , applied your suggestion. Anything else, or is this ready to merge?

**Review by marefr — APPROVED** [review]
LGTM

**Pull request merged**